index.html

```diff
@@ -46,31 +46,59 @@
       var unityInstance = UnityLoader.instantiate("unityContainer", "Build/YouVsVirus_WebGL_v0.1_15-04-2020.json", {onProgress: UnityProgress});
     </script>
 <body>
-
-      <nav class="navbar navbar-expand-lg navbar-dark ftco_navbar bg-dark ftco-navbar-light" id="ftco-navbar">
+ <!--
+      <nav class="navbar navbar-dark bg-dark>
         <div class="container">
           <a class="navbar-brand" href="index.html">YouVsVirus</a>
           <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#ftco-nav" aria-controls="ftco-nav" aria-expanded="false" aria-label="Toggle navigation">
             <span class="oi oi-menu"></span> Menu
           </button>
 
           <div class="collapse navbar-collapse" id="ftco-nav">
-            <!--   <ul class="navbar-nav ml-auto">
-                <li class="nav-item active"><a href="index.html" class="nav-link">Home</a></li>
-            </ul>-->
+                <ul class="navbar-nav ml-auto style="white"">
+                 <li class="nav-item active"><a href="index.html" class="nav-link">Home</a></li>
+                <li class="nav-item active"><a href="index.html" class="nav-link">Impressum</a></li>
+            </ul>
           </div>
         </div>
       </nav>
+ -->
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+  <a class="navbar-brand" href="#">YouVsVirus</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarText">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item active">
+        <a class="nav-link" href="index.html">Home <span class="sr-only">(current)</span></a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="impressum.html">Impressum</a>
+      </li>
+    </ul>
+    <span class="navbar-text">
+    Stay home.
+    </span>
+  </div>
+</nav>
+
+
     <!-- END nav -->
     
 
 
```

Comment:
All these changes below this line should not be included in this pull request.
They revert recent changes on the master (margrit/release).
I think merging master into this commit and cleaning this up should resolve it.

Reply:
done